test.cpp

```diff
@@ -235,6 +235,36 @@ std::map<std::string, void (*)()> constuct_tests() {
    EXPECT(x.data().min().toCFloat() == 0);
  };
 
+ tests["autograd/embedding/basic"] = []() {
+   int dict_size = 10;
+   auto model = Embedding(dict_size, 2).make();
+   // Cannot get gradients to change indices (input) - only for embedding params
```

Comment:
What does this comment mean?

Reply:
The input to Embedding will be an index e.g. "2", which is discrete/non-differentiable (e.g. can't have an index of 2.1). This is why `requires_grad` for `x` is false.